lib/config.yml

```diff
@@ -310,6 +313,9 @@ Style/GuardClause:
 Style/HashEachMethods:
   Enabled: true
 
+Style/HashSyntax:
+  EnforcedShorthandSyntax: never
```

Comment:
https://rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashSyntax

Not setting the shorthand property here means ~17,000 rubocop errors. We could use autocorrect to get most of them, but that also does a good job at making the git history less useful.

I set it to `never` here for the same reason as above (this overly terse Ruby syntax is not helping code readability IMO), but this could also be set to `either` to let folks just use whatever they want.

Reply:
I would prefer `either` here. Ever since i learned that we would be able to do `{foo:, bar:}` instead of `{foo: foo, bar: bar}` I've been waiting for this. To me this is a still-readable convenient shorthand, compared to `&` to pass blocks in the previous which seems less clear.

That said, it's not the end of the world if others really don't want this. At the risk of a debate apocalypse, may I suggest a vote on `#dev` to make these a truly democratic decision.

Reply:
I updated this to `either`. I agree, it's not worth the inevitable endless debate.